hw_9.py

```diff
@@ -0,0 +1,67 @@
+# 1. Використати файл як базу даних для збереження записів телефонної книги із попередніх завдань.
+import json
```

Comment:
Your phonebook should be the same as in the homework where you created it initially. There should be only a few updates:
- initialize the phonebook as a dict by reading the data from a file and loading it from json format
- rewrite a file when you make some changes to the phonebook dict (command add and delete)